segmentation/include/pcl/segmentation/impl/lccp_segmentation.hpp

```diff
@@ -360,10 +358,8 @@ pcl::LCCPSegmentation<PointT>::applyKconvexity (const unsigned int k_arg)
     return;
 
   EdgeIterator edge_itr, edge_itr_end, next_edge;
-  boost::tie (edge_itr, edge_itr_end) = boost::edges (sv_adjacency_list_);
-
   // Check all edges in the graph for k-convexity
-  for (next_edge = edge_itr; edge_itr != edge_itr_end; edge_itr = next_edge)
+  for (std::tie (edge_itr, edge_itr_end) = boost::edges (sv_adjacency_list_), next_edge = edge_itr; edge_itr != edge_itr_end; edge_itr = next_edge)
   {
     next_edge++;  // next_edge iterator is necessary, because removing an edge invalidates the iterator to the current edge
```

Comment:
Let's use `++next_edge` here and L418 (in this PR or another). LGTM otherwise.

Reply:
Done.